lib/auth/init.go

```diff
@@ -414,20 +414,32 @@ func migrateRoles(asrv *AuthServer) error {
 		return trace.Wrap(err)
 	}
 
-	// loop over all roles and only migrate RoleV2 -> RoleV3
+	// DEPRECATED: Remove this migration in Teleport 2.5.
```

Comment:
I suggest to use tag: `DELETE IN: 2.5.0` as a first line and then second line should explain why.